build/repo.props

```diff
@@ -51,7 +51,7 @@
 
     <!-- These projects are meant to be executed by tests. -->
     <ProjectToExclude Include="
-                      $(RepositoryRoot)src\Tools\dotnet-watch\test\TestProjects\**\*.csproj;
+                      $(RepositoryRoot)src\Tools\Tests.Common\TestProjects\**\*.csproj;
```

Comment:
Nit: put in a folder named testassets. This is a shared test library, not a new package, so needs to be categorized as a test asset. This happens automatically if the parent folder is named testassets.